addons/mail/static/src/new/web/chatter.xml

```diff
@@ -140,7 +140,7 @@
 </t>
 
 <t t-name="mail.chatter.add_attachment_button" owl="1">
-    <button class="o-mail-chatter-topbar-add-attachments btn btn-light" t-att-class="{ 'my-2': !props.compactHeight }" t-att-disabled="!props.resId" t-on-click="() => state.isAttachmentBoxOpened = !state.isAttachmentBoxOpened">
+    <button class="o-mail-chatter-topbar-add-attachments btn btn-light" aria-label="Attach files" t-att-class="{ 'my-2': !props.compactHeight }" t-att-disabled="!props.resId" t-on-click="() => state.isAttachmentBoxOpened = !state.isAttachmentBoxOpened">
```

Comment:
This `aria-label` might be misleading depending on current state.

It's only "Attach files" when there are currently no attachments, otherwise it is either open or close the attachment box depending on its state.

Reply:
Agree it's a bit misleading. I'm not sure it's worth having a dedicated wording for toggling. Might become too technical. "Attach files" gives a good enough intention for both behaviours